### PR TITLE
Create Adb

### PR DESCRIPTION
**Pull request by Savag353aba55**

### CODE REVIEW

File: Adb [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Himniharvest) [inline Adb:1]:
```suggestion
[
```

**Review by Himniharvest — APPROVED** [review]

**Comment by charlykt** [comment]
https://patch-diff.githubusercontent.com/raw/jarhot1992/Remote-ADB/pull/40.diff

**Review by TA20231980 — CHANGES_REQUESTED** [review]
- [ ] - @Savag353aba55

File: Adb (hunk 1 above)
Comment (Priyadiwidjaja929269) [reply Adb:1]: ❤️‍🩹❤️‍🩹

Comment (Priyadiwidjaja929269) [reply Adb:1]: ❤️‍🩹❤️‍🩹

**Pull request closed without merging**